### PR TITLE
Chschrae/update to sdk1.8

### PR DESCRIPTION
**Pull request by chschrae**
## Purpose

This change updates the Speech SDK to the latest version 1.8 and associated UI changes

## Does this introduce a breaking change?

```
[x ] Yes
[ ] No
```
previous Direct line speech bots will need to register with the cognitive services subscription key in order for this new version to work
DLS1 8Main
DLS1 8Settings

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ x] Other... Please describe: Update to the SDK and UI
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: DLSpeechClient/MainWindow.xaml.cs [hunk 1]
```diff
@@ -189,63 +187,59 @@ private void InitSpeechConnector()
...
-            // Save the Direct Line Speech channel secret key. This is one of two keys you get when you register your bot with Direct Line speech
-            // channel. It uniquely defines the bot. Here we call it bot secret for short.
-            this.botSecret = this.botSecretLabel.Text;
-
-            if (!string.IsNullOrWhiteSpace(this.settings.Settings.SubscriptionKey) &&
-                !string.IsNullOrWhiteSpace(this.botSecret))
+            if (!string.IsNullOrWhiteSpace(this.settings.RuntimeSettings.SubscriptionKey) &&
+                !string.IsNullOrWhiteSpace(this.settings.RuntimeSettings.SubscriptionKeyRegion))
             {
                 // Set the dialog service configuration object based on three items:
```
Comment (dargilco) [inline DLSpeechClient/MainWindow.xaml.cs]: two items

File: DLSpeechClient/MainWindow.xaml.cs [hunk 2]
```diff
@@ -189,63 +187,59 @@ private void InitSpeechConnector()
...
-                // - Direct Line Speech channel secret (aka "bot secret")
```
Comment (dargilco) [inline DLSpeechClient/MainWindow.xaml.cs]: remove this line

File: DLSpeechClient/MainWindow.xaml.cs (hunk 1 above)
Comment (chschrae) [reply DLSpeechClient/MainWindow.xaml.cs]: I don't understand what you mean here...

File: DLSpeechClient/MainWindow.xaml.cs (hunk 2 above)
Comment (chschrae) [reply DLSpeechClient/MainWindow.xaml.cs]: ah good catch

File: DLSpeechClient/MainWindow.xaml.cs (hunk 1 above)
Comment (chschrae) [reply DLSpeechClient/MainWindow.xaml.cs]: Oh I think I see it now. git flagged different lines

**Pull request merged**